src/inverseRobot.js

```diff
@@ -8,6 +8,21 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inverseArr = {};
```

Comment:
```suggestion
  const inversedArr = {};
```
or even better - inversedRobot

Reply:
i feel so stupid right now,.. i was on this so many hours and the solution was so obvious...  :( thank you sensei !